packages/pciutils/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "pciutils"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+[package.metadata.build-package]
+releases-url = "https://mj.ucw.cz/download/linux/pci/"
```

Comment:
Nit: pciutils is also available at  https://www.kernel.org/pub/software/utils/pciutils/. Should we use that for consistency? We have a good number of packages that consume from kernel.org.

Reply:
Good catch, I had switched in the spec file and not here.